client/src/components/generic/EventsCard/EventsCard.tsx

```diff
@@ -1,6 +1,8 @@
 import Button from "../FigmaButtons/FigmaButton"
-
-interface EventsCardProps {
+/**
+ * Props for event card
+ */
+export interface EventsCardProps {
   date?: string
```

Comment:
i meant to annotate each of the individual props

Reply:
done